src/Tools/DynamoFeatureFlags/DynamoFeatureFlags.csproj

```diff
@@ -11,6 +11,13 @@
     <PackageReference Include="CommandLineParser" Version="2.8.0" />
     <PackageReference Include="LaunchDarkly.ClientSdk" Version="2.0.1">
     </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
```

Comment:
shouldn't this package be under a net6 condition ?

Reply:
or do we want to use it for net48 ?
That would mean an extra dependency for net48

Reply:
this package works for both net48 and net6.0, it shouldn't be an issue to use this package, currently the xml config file is  being parsed manually

Reply:
We have a policy to not bring in as few extra dependencies as possible.
So if this is not needed for net48 then please limit it to net6 only